index.html

```diff
@@ -169,7 +174,7 @@ <h2>Contact me</h2>
                         <input name="username" type="username" placeholder="lorem lpsum" maxlength="30" required>
                         <input name="email" type="email" placeholder="loremipsum@gmail.com" required>
                         <textarea name="textarea" maxlength="500" placeholder="write your message here.."></textarea>
-                        <button name="button" type="submit" id="btn-f">Get in touch</button>
+                        <button name="button" type="submit">Get in touch</button>
```

Comment:
- [ ] Great job on your contact form section! I just wanted to offer **a small suggestion to help make it even more consistent with the given [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34-848&t=9T0c1LaQy7Q424QS-0)**. Specifically, **you may want to double-check the width of your `Get in touch` button for the desktop version.**
By ensuring that the button width matches the desired template, you can create a more polished and professional appearance that will be consistent across all devices and screen sizes. 🙌

  ### Your button:
  ![image](https://user-images.githubusercontent.com/107952724/236502653-594b8780-b823-433b-b5a0-bf53c8c2a930.png)

  ### Figma's button:
  ![image](https://user-images.githubusercontent.com/107952724/236502753-01c39350-4f5d-4bfe-9bd0-06563c1ec9b2.png)